eng/common/pipelines/templates/steps/get-pr-owners.yml

```diff
@@ -1,16 +1,30 @@
 parameters: 
   TargetVariable: ''
+  TargetUserVariable: 'notspecified'
```

Comment:
These should default to empty and if they are empty then we don't write any variable.